celesta-maven-plugin/src/main/java/ru/curs/celesta/plugin/maven/CursorGenerator.java

```diff
@@ -314,6 +317,21 @@ private static String calcClassName(GrainElement ge, String sourceFileNamePrefix
         return Arrays.asList(grainName, objectName);
     }
 
+    private static List<FieldSpec> buildColumnFields(DataGrainElement ge) {
+
+        Map<String, ? extends ColumnMeta> columns = ge.getColumns();
+
+        return columns.entrySet().stream()
+                .filter(e -> e.getValue().getJavaClass() != BLOB.class)
+                .map(e -> FieldSpec.builder(
+                        ParameterizedTypeName.get(ColumnRef.class, e.getValue().getJavaClass()),
+                        e.getKey() + "_COLUMN",
```

Comment:
Instead of **_COLUMN** one could have **_REF**, **_FIELD** or something else. I'm biased, but I think that **_COLUMN** is not the best choice either.

Reply:
In generated classes it looks rather ugly and not Java style. Why not just add `...Column` instead, like `...Cursor`?